starknet_devnet/blueprints/rpc/structures/types.py

```diff
@@ -113,5 +113,5 @@ class PredefinedRpcErrorCode(Enum):
 RPC_ERRORS = json.loads(RPC_SPECIFICATION)["components"]["errors"]
 
 GATEWAY_TO_RPC_ERROR = {
-    StarknetErrorCode.OUT_OF_RANGE_BLOCK_ID: RPC_ERRORS["BLOCK_NOT_FOUND"],
```

Comment:
Why change this? Shouldn't we just add a new entry?

Reply:
After the last changes, this code was not working as expected on `get` with `ex.code` in `rpc_error_dict`:
```
except StarknetDevnetException as ex:
        default_error = PredefinedRpcErrorCode.INTERNAL_ERROR
        rpc_error_dict = GATEWAY_TO_RPC_ERROR.get(ex.code, default_error)
        return rpc_error(message_id=message_id, **rpc_error_dict)
```
and it's the only one place where `GATEWAY_TO_RPC_ERROR` is used so I decided to change this